setup.py

```diff
@@ -1,21 +1,23 @@
 # -*- coding: utf-8 -*-
 from setuptools import setup
 
+version = '0.8.7'
+homepage = 'https://github.com/python-amazon-mws/python-amazon-mws'
 short_description = 'Python library for interacting with the Amazon MWS API'
 
 try:
     from pypandoc import convert_file
     long_description = convert_file('README.md', 'rst')
 except (ImportError, OSError):  # either pypandoc or pandoc isn't installed
-    long_description = "See README.md"
+    long_description = 'See README.md'
 
 setup(
     name='mws',
-    version='0.8.6',
-    maintainer="James Hiew",
-    download_url='https://github.com/python-amazon-mws/python-amazon-mws/archive/v0.8.6.tar.gz',
-    maintainer_email="james@hiew.net",
-    url="http://github.com/jameshiew/mws",
+    version=version,
+    maintainer='python-amazon-mws',
+    download_url=homepage + '/archive/v{}.tar.gz'.format(version),
+    maintainer_email='james@hiew.net',
```

Comment:
I've setup a google group for this.

python-amazon-mws@googlegroups.com

I've invited both you and @GriceTurrble so far.